samples/nrf9160/tcp_client/Kconfig

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "TCP Sample Settings"
+
+config TCP_DATA_UPLOAD_SIZE_BYTES
+	int "Number of bytes transmitted to the server"
+	default 10
+
+config TCP_DATA_UPLOAD_FREQUENCY_SECONDS
+	int "How often data is transmitted to the server"
+	default 900
+
+config TCP_SERVER_ADDRESS_STATIC
+	string "TCP server IP address"
+	default "8.8.8.8"
+
+config TCP_SERVER_PORT
+	int "TCP server port number"
+	default "2469"
```

Comment:
Out of curiosity, what service is underneath this port number? Is it documented somewhere that it's open at Google's DNS server?

Reply:
do not know the story behind it, just used the same port as in udp sample

Reply:
And does it actually work for you? I failed to connect with `qemu_x86` (can't test with LTE right now as I apparently have some connectivity issues). Also `nmap` scan did not show that his port is open at `8.8.8.8`.

Reply:
For the UDP sample, I think this port number (the height of Norway's highest mountain, coincidental or not) was chosen because it's most likely not listened to. The aim was to measure power consumption when sending data, not receiving. Not sure if that assumption holds though, I'd expect ICMP "destination unreachable" packets in return, but that's not relevant for this PR.

For TCP I think we'd want to use an IP address and port number that allows TCP connections, so that we get a realistic picture of power consumption with ACKing all that.

Reply:
I have not yet found a viable public IP address and port to use for testing.

So maybe I will follow @simensrostad suggestion of  including in the documentation how to set up your own TCP endpoint to test with. (e.g. TCP echo server)

Reply:
I used to test with http://portquiz.net, but I'm not sure how stable this service is.